src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -116,9 +116,9 @@ public async Task<ActionResult<List<DataFileInfo>>> GetDataFileInfo(Guid release
         [ProducesResponseType(404)]
         [RequestSizeLimit(int.MaxValue)]
         [RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = int.MaxValue)]
-        public async Task<ActionResult<DataFileInfo>> AddDataFilesAsync(Guid releaseVersionId,
+        public async Task<ActionResult<DataFileInfo>> Upload(Guid releaseVersionId,
```

Comment:
> I think the name Upload doesn't give much away in the context of the ReleasesController. How about UploadDataSet? Similar comment regarding the other data set file operations in this controller.

Renamed the Upload* methods in `ReleasesController`. I've not renamed the `releaseDataFileService` methods - but can if you think that's worthwhile.

> Did you consider moving data set file methods out to a new controller because it's already pretty busy with core release methods?

I may have done? :)

I'll leave them for now, but just say if you want them moved.

Reply:
I think that one's ok to leave. 🙂